.github/workflows/test-suite.yaml

```diff
@@ -114,7 +180,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        etest: [dnscache, kine]
+        etest: [dnscache, kine, ciliumnokp, multus]
```

Comment:
is this a miss-spelled cilium ?

Reply:
No, its a shorthand for "cilium no kube-proxy".